dolphinscheduler-ui/src/views/projects/task/instance/batch-task.tsx

```diff
@@ -74,20 +77,27 @@ const BatchTaskInstance = defineComponent({
       variables.showModalRef = false
     }
 
-    const getLogs = (row: any) => {
+    const getLogs = (row: any, logTimer: number) => {
       const { state } = useAsyncState(
         queryLog({
           taskInstanceId: Number(row.id),
           limit: variables.limit,
           skipLineNum: variables.skipLineNum
         }).then((res: any) => {
-          if (res?.message) {
-            variables.logRef += res.message
+          variables.logRef += res.message || ''
+          if (res && res.message !== '') {
             variables.limit += 1000
             variables.skipLineNum += res.lineNum
-            getLogs(row)
+            getLogs(row, logTimer)
           } else {
             variables.logLoadingRef = false
+            setTimeout(() => {
+              variables.logRef = ''
+              variables.limit = 1000
+              variables.skipLineNum = 0
+              variables.logLoadingRef = true
+              getLogs(row, logTimer)
+            }, logTimer * 1000)
```

Comment:
It's better to clear the timer before turning it on.

Reply:
Can you elaborate on this comment? 
I tried clearing the timer by `clearTimeout(timeoutID)` but it might not be what we want.

Reply:
If the `getLogs` function is called multiple times, multiple timers will be running.

Reply:
https://github.com/apache/dolphinscheduler/pull/11226/commits/7782c5d6cf93a4473fcdf8288ee433ed61870388